File: 0455-assign-cookies/0455-assign-cookies.py
class Solution:
    def findContentChildren(self, g: List[int], s: List[int]) -> int:
        greed=0
        cookie=0
        g.sort()
        s.sort()
        while(greed<len(g) and cookie<len(s)):
            if(s[cookie]>=g[greed]):
               greed+=1
            cookie+=1
            
        return greed

File: 0455-assign-cookies/0455-assign-cookies.cpp
class Solution {
public:
    int findContentChildren(vector<int>& g, vector<int>& s) {
        sort(g.begin(), g.end());
        sort(s.begin(), s.end());
        int i=0;
        int j=0;
        while(j<s.size() && i<g.size()){
            if(s[j]>=g[i])
                i++;
            j++;
            }
        return i;
    }
};